panel/param.py

```diff
@@ -95,9 +96,9 @@ def __init__(self, object, **params):
             params['name'] = object.name
         super(Param, self).__init__(object, **params)
         self._widgets = self._get_widgets()
-        self._widget_box = WidgetBox(*self._widgets.values(), height=self.height,
+        widgets = [widget for widgets in self._widgets.values() for widget in widgets]
+        self._widget_box = WidgetBox(*widgets, height=self.height,
                                      width=self.width, name=self.name)
-
         kwargs = {'name': self.name}
         if self.subpanel_layout is Tabs:
```

Comment:
Should "subpanel" still be appearing in the source code?  I would think it should say "pane" or "viewable"?

Reply:
Neither seem very descriptive, it's specifically about the layout for the parameterized subobjects. I guess `subobject_layout` would work.

Reply:
That's better for this case, thanks.